services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(id=movie_session_id)
```

Comment:
Try to extract the required information here regarding `row` and `seat` from all tickets using `values` or `values_list`. It will execute a query to extract only required values and will not query for objects as a whole.